src/render.rs

```diff
@@ -218,6 +226,31 @@ impl<'a> Context<'a> {
         )?;
 
         let command_buffers = init_commands(&drawing_pool, &framebuffers)?;
+        let font_path = config::load_section_setting::<String>(
```

Comment:
This all probably belongs a bit higher up, with a `/* Text */` header or something (should be above debug and the command buffers at least)